src/main/java/it/pagopa/transactions/client/PaymentGatewayClient.java

```diff
@@ -54,14 +54,15 @@ private Mono<PostePayAuthResponseEntityDto> requestPostepayAuthorization(Authori
                 .switchIfEmpty(Mono.empty())
             .map(authorizationRequestData ->
                 new PostePayAuthRequestDto()
-                    .grandTotal(BigDecimal.valueOf(((long) authorizationData.transaction().getAmount().value()) + authorizationData.fee()))
-                    .description(authorizationData.transaction().getDescription().value())
+                    .grandTotal(BigDecimal.valueOf(((long) authorizationData.transaction().getNoticeCodes().stream().mapToInt(noticeCode -> noticeCode.transactionAmount().value()).sum()) + authorizationData.fee()))
+                    .description(authorizationData.transaction().getNoticeCodes().get(0).transactionDescription().value())
```

Comment:
TODO handle multiple description due to multiple notice code

Reply:
Task opened